.github/workflows/build-images.yml

```diff
@@ -196,32 +196,25 @@ jobs:
       image: ${{ steps.legacy_vertica_image.outputs.value }}
     steps:
 
-    - name: set lower case owner name
-      env:
-        OWNER: '${{ github.repository_owner }}'
-      run: |
-        echo "OWNER_LC=${OWNER,,}" >>${GITHUB_ENV}
-
     - name: Pick the name of the image
       uses: spilchen/switch-case-action@v2
       id: legacy_vertica_image
       with:
-        default: ghcr.io/${{ env.OWNER_LC }}/vertica-k8s:${{ github.sha }}-legacy
+        default: docker.io/opentext/vertica-k8s-private:latest-legacy-master
```

Comment:
What's the difference between the sha-legacy and the latest-legacy-master image?

Reply:
sha-legacy image will be built by e2e tests, and latest-legacy-master is built by devops and already exists before e2e tests run.